chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -61,11 +61,40 @@
   include_recipe "ceph::radosgw_keystone"
 end
 
+rgw_conf = "ceph.conf"
+
+if node["platform_family"] == "suse"
+  rgw_conf = "ceph.conf.radosgw"
+  # When generating the override systemd unit file, we have to disable the
+  # radosgw service so that it gets re-enabled later, thus picking up the
+  # override unit file (if we didn't do this, a host that was already
+  # running radosgw and was then upgraded wouldn't pick up the new unit file).
+  # (see also the comments about this in conf.rb)
+  # The first sed strips any existing --conf option out of the ExecStart line
+  # (this was present in Hammer, but is no longer present in Jewel).  The
+  # second sed tells radosgw to explicitly use our radosgw specific conf file.
+  bash "generating override ceph-radosgw systemd unit file" do
+    code <<-EOH
+      sed -e 's%^\\(ExecStart=.*\\)\\(--conf [^ ]*\\)\\(.*\\)%\\1\\3%' \
+        /usr/lib/systemd/system/ceph-radosgw@.service | \
+      sed -e 's%^\\(ExecStart=.*\\)%\\1 --conf /etc/ceph/ceph.conf.radosgw%' \
```

Comment:
You may well be right (and also with the `--in-place` option). I spent some time testing variants of this, but I honestly can't remember if I tried combining the two or not. Speaking purely from a selfish point of view, I'd rather it just go in as-is if you don't have a strong objection, because it'll save me a couple of rounds of further testing ;-)